nuxeo-features/nuxeo-platform-tag-service/nuxeo-platform-tag-core/src/main/java/org/nuxeo/ecm/platform/tag/TagServiceImpl.java

```diff
@@ -36,7 +36,7 @@
     protected TagService tagService;
 
     @Override
-    public void start(ComponentContext context) {
+    public void activate(ComponentContext context) {
```

Comment:
I really don't like that we have to do this, it's going backward. Couldn't we just make sure the getApplicationStartedOrder of the TagService is before the one of the RepositoryService?

Reply:
Right, and besides it will prevent having the tag service depending on the component defining the configuration property.